app/src/main/java/eu/kanade/tachiyomi/App.kt

```diff
@@ -220,8 +220,8 @@ class App : Application(), DefaultLifecycleObserver, SingletonImageLoader.Factor
             // Override the value passed as X-Requested-With in WebView requests
             val stackTrace = Looper.getMainLooper().thread.stackTrace
             val isChromiumCall = stackTrace.any { trace ->
-                trace.className.equals("org.chromium.base.BuildInfo", ignoreCase = true) &&
-                    setOf("getAll", "getPackageName", "<init>").any { trace.methodName.equals(it, ignoreCase = true) }
+                setOf("org.chromium.base.BuildInfo", "org.chromium.base.ApkInfo").any { trace.className.equals(it, ignoreCase = true) } &&
+                setOf("getAll", "getPackageName", "<init>").any { trace.methodName.equals(it, ignoreCase = true) }
```

Comment:
Looking at the code `ApkInfo` should only have an associated `<init>` call. We can probably simplify this a bit based on that. It's still fine as is. Also add a entry in CHANGELOG.md